tables/hbo-slim.utb

```diff
@@ -0,0 +1,100 @@
+# liblouis: Classical/Biblical Hebrew with no shewa or dagesh (Katz)
+#
+#-index-name: Hebrew, classical, slim
+#-display-name: Classical Hebrew without shewa or dagesh
+#
+#+language: hbo
+#+language: oar
+#+language: arc
+#+language: jpa
+#+language: obm
+#+language: xdm
+#+language: phn
+#+type: literary
+#+contraction: yes
+#+system: Katz
+#+dots: 6
+
+#-license: lgpl-2.1
+
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+# The first Hebrew Braille table hosted on Lib Louis was developed by
```

Comment:
I sound like a nag, but the official terminology is "liblouis", not "Lib Louis" :-)